buildbot/build_tflite.sh

```diff
@@ -127,10 +145,30 @@ ninja -C gemmlowp/src/gemmlowp-build install
 git clone --filter=tree:0 --no-checkout ${ML_DTYPES_REPOSITORY} ml_dtypes/src/ml_dtypes
 git -C ml_dtypes/src/ml_dtypes checkout ${ML_DTYPES_TAG}
 
+# pthreadpool
+# NOTE: currently a hack, because pthreadpool doesn't support find_package.
+# we install in the default install dir, which really means the script must be run
+# under sudo. Works for buildbots. Not ideal elsewhere.
```

Comment:
This would need doc updates probably?

I'm wondering if we should just install it to the sysroot if that's specified, but otherwise default to the system dependency.